lib/cli/run.js

```diff
@@ -258,6 +258,13 @@ exports.builder = yargs =>
         }
       });
 
+      types.boolean
+        .concat(types.string, types.number)
+        .filter(opt => Array.isArray(argv[opt]))
+        .forEach(opt => {
+          argv[opt] = argv[opt].pop();
+        });
+
```

Comment:
This part does not work correctly:
- input per CLI: `--slow 5 --slow 10`
- result after parsing: `s: ["5"], slow: "10"`